src/managers/GroupManager.ts

```diff
@@ -82,6 +85,44 @@ export default class GroupManager extends BaseManager<Group> implements GroupMan
         return group
     }
 
+    /**
+     * Joins a group.
+     *
+     * @param inviteLinkOrGroupID The group invite link or group ID.
+     * @param shareToken The group's share token.
```

Comment:
isn't it possible to document these overloads separately so that users aren't faced with this confusing `inviteLinkOrGroupID` thing? it's better if the user sees only either ( `inviteLink` ) or ( `groupID` and `shareToken` ) rather than this cursed amalgamation of the two which we have to deal with internally, so that it's clear that we aren't expecting an invite link for the first parameter and a share token for the second